db/schema.rb

```diff
@@ -61,6 +61,20 @@
     t.datetime "updated_at"
   end
 
+  create_table "advice_plans_advice_plans", force: :cascade do |t|
+    t.string   "code",         limit: 255,                 null: false
+    t.boolean  "current",                  default: false, null: false
+    t.datetime "started_at"
+    t.datetime "completed_at"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  create_table "advice_plans_advice_plans_tasks", id: false, force: :cascade do |t|
+    t.integer "advice_plan_id", limit: 4, null: false
+    t.integer "task_id",        limit: 4, null: false
+  end
+
```

Comment:
I didn't understand why these tables appear on this commit? Were they deleted before and re-appear again on the schema.rb?

Reply:
Me neither, I need to sort out the schema before merging, but its difficult to do remotely so if you want to pull the branch, feel free.

Reply:
I will check the status of the schema.rb and see what happened.

Reply:
Thanks @tomas-stefano